lib/package.json

```diff
@@ -1,27 +1,32 @@
 {
-  "name": "react-ssd1306-app",
+  "name": "react-moddable",
   "version": "0.0.0",
-  "description": "React SSD1306 Demo App",
+  "description": "React Moddable experiments",
   "main": "index.js",
   "private": true,
   "scripts": {
-    "dev-web": "parcel src/index.html",
     "dev": "rollup -c build/rollup.config.js --watch",
     "build": "rollup -c build/rollup.config.js"
   },
   "keywords": [],
-  "author": "Yifeng Wang",
+  "author": "Shinya Ishikawa",
```

Comment:
😰

Reply:
Oh that's my bad. I think when I copy&pasted the package configuration from my another repository I've overriten it. I will revert the author name ASAP. Apologies and thank you for pointing it out.

Reply:
I removed the change on my fork.
https://github.com/meganetaaan/react-moddable/blame/main/lib/package.json#L12

Reply:
Hi @meganetaaan, you can certainly use your name as the author on a heavily modified fork like this, as long as you don't merge it back upstream (it would be even better if your README came with something like a "based on XXX" credit). I have no intention of asking you to modify your fork version, since this looks like just a misunderstanding that you accidentally submitted a PR to upstream (which is why I got notified). In other words I'm just commenting on the unexpected PR, not the code you forked.

Also, I'm excited to see someone continue to evolve React on embedded systems, happy hacking!